src/cpp/session/modules/SessionHelp.cpp

```diff
@@ -125,6 +125,21 @@ bool isLocalURL(const std::string& url,
    return false;
 }
 
+std::string normalizeHttpdSearchContent(const std::string& content)
+{
+   return boost::regex_replace(
+            content,
+            boost::regex("(The search string was <b>\")(.*)(\"</b>)"),
+            [](const boost::smatch& m)
+   {
+      std::string query = m[2];
```

Comment:
What's the purpose of the `query` variable ? (it isn't used and seems like we *always* should `htmlEscape` the search string, yes?)

Reply:
The intention was to avoid double-escaping the search string (e.g. if the behavior of R were to change in the future)